.github/ISSUE_TEMPLATE/user_story.md

```diff
@@ -0,0 +1,25 @@
+---
+name: "User Story \U0001F4DA"
+about: Bundle multiple tasks/ideas into one user story
+labels: Story
+---
+
+### Describe the Problem
+
+As a user I want XYZ such that XYZ.
+
+### Acceptance Criteria
+
+- [ ] Criteria 1...
+
+### Technical Tasks
+
+- [ ] Task 1...
```

Comment:
Maybe we can use the new sub-issues feature instead? This could help to create smaller chunks of work which may also be worked on in parallel.

Reply:
As discussed I'll remove this template and add a note to the task template.